lib/class-wp-rest-sidebars-controller.php

```diff
@@ -351,6 +351,17 @@ public static function get_widgets( $sidebar_id ) {
 						$widget['id_base']      = $instance->id_base;
 					}
 
+					if ( $request['context'] === 'edit' && isset( $wp_registered_widget_controls[ $widget_id ]['callback'] ) ) {
```

Comment:
I don't like checking for `context` here, but that's a nested field and the core only filters top-level field. Seems like a opportunity for bugfix/new feature :-)

Reply:
We also should really be moving this to a `/widgets` controller ASAP.

Reply:
Spin this in another issue? I would love this `/widgets` endpoint 👍

Reply:
Another issue works!

Reply:
https://github.com/WordPress/gutenberg/issues/25232